```
tools/cabana/messageswidget.cc:80:3: error: unknown type name 'QFile'
  QFile json_file("./car_fingerprint_to_dbc.json");
```
I think I have the same issue on Ubuntu 22.04

--- a/tools/cabana/messageswidget.cc
+++ b/tools/cabana/messageswidget.cc
@@ -2,13 +2,13 @@
 
 #include <QCompleter>
 #include <QDialogButtonBox>
+#include <QFile>
 #include <QFontDatabase>
 #include <QHeaderView>
 #include <QLineEdit>
 #include <QPushButton>
 #include <QTextEdit>
 #include <QVBoxLayout>
-#include <QFile>
 
 #include "tools/cabana/dbcmanager.h"
 

